### DISCUSSION TITLE
The newly added "inpaint" feature works well

### DISCUSSION THREAD

**Discussion by Timzoid** [asigalov61/Allegro-Music-Transformer#5] [Ideas]
I did about six inpaints of two small Bartok pieces in Allegro this morning, same ones I did yesterday in LAMC, and they seem to be good -- very pleased with it. It'll take me a few weeks to decide which one I'll use for inpainting regularly. Thanks for adding it though, as I was curious how it would be, and I am using Allegro, Composer Version more now. 

I also tried one of your other programs, Orchestrator. Besides not being able to choose instruments really, it seems a little arbitrary in the way it switches instruments. One of its faults is it flattens the dynamic expression of the seed. 

Before the newly added instruction to "Press the play button..." to load the seed, it might be good if you could put an arrow pointing to the left:
 <--  Press play button...

The reason I think that might be helpful is although the button to run cells is the universal play button symbol, there might be a little confusion in that "play" is almost always associated with playing music, and indeed, you have another play button to click on to review a sequence. Adding the arrow would save people from looking around for some other play button. 

Your instruction on the instruments choice for the "inpaint" feature is clear now. People, including myself, used to MuseNet or many other such programs, just click on the instruments they want for the finished sequence. It makes total sense that the feature for "inpaint" is about choosing the instruments the sequence is already in.

**Comment by asigalov61** [comment]
@Timzoid You are welcome about inpainting generator :)

I will see if I can add an arrow to the notes on custom MIDI sections in my implementations, but it will have to wait a bit till the next code/colab updates.

I am glad to hear that inpainting notes are understandable and clear. This is good.

Thank you for trying it all out and for your feedback. Much appreciated :)

Alex.
